perfkitbenchmarker/openstack/os_virtual_machine.py

```diff
@@ -0,0 +1,155 @@
+import logging
+import random
+import string
+import time
+
+from perfkitbenchmarker import virtual_machine, linux_virtual_machine
+from perfkitbenchmarker import flags
+from perfkitbenchmarker import vm_util
+from perfkitbenchmarker.openstack import utils as os_utils, os_disk
+
+
+FLAGS = flags.FLAGS
+
+flags.DEFINE_string('os_auth_url', 'http://localhost:5000',
```

Comment:
The flags are potentially confusing - all flags are in a global namespace even when only used in a specific cloud provider, and there's already an unrelated `--os_type` flag.

I think it would be clearer to use "openstack" as a prefix instead of "os".